assets/react/controllers/Mosaic.jsx

```diff
@@ -42,7 +42,11 @@ function Mosaic() {
         const errorMessageFailedToFetch = "Could not fetch tiles";
         const randomPath = config.randomTiles ? '/random' : "";
 
-        fetch(`/api/v1/tiles${randomPath}?` + new URLSearchParams({
+        const screenTags = screen.tags.map(tag =>
```

Comment:
It should be possible to write this with URLSearchParams.append
https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/append

```suggestion
        const searchParams = new URLSearchParams({
            page: 1,
            limit: config.numberOfTiles,
            'order[updatedAt]': 'desc',
         });
         
         screen.tags.forEach((tag) => {
           searchParams.append("tags.tag", tag.tag);
         });
         
         fetch(`/api/v1/tiles${randomPath}?` + searchParams
         ....
```

Reply:
Thanks for the suggestion. It's way cleaner that way.